src/distributions/microsoft/installer.ts

```diff
@@ -45,6 +45,10 @@ export class MicrosoftDistributions extends JavaBase {
       throw new Error('Early access versions are not supported');
     }
 
+    if (this.packageType !== 'jdk') {
```

Comment:
Good catch here.

Rather than raising an error, can we just ignore the `java-package` input and always use `jdk`?  If a different `java-package` input is passed, we can print a warning:

```
Microsoft Build of OpenJDK always uses the `jdk` package type.  Remove the `java-package` argument when calling the action to silence this warning.
```

Reply:
Done. Thank you.